hpx/runtime/threads/coroutines/detail/context_linux_x86.hpp

```diff
@@ -140,7 +151,19 @@ namespace hpx { namespace threads { namespace coroutines
 
             x86_linux_context_impl()
                 : m_stack(nullptr)
-            {}
+            {
+                segv_stack.ss_sp = valloc(SEGV_STACK_SIZE);
```

Comment:
We usually name our cmake config variable `HPX_WITH_<FOO>` and the corresponding preprocessor constant `HPX_HAVE_<FOO>`.

Also, `hpx::cerr` might not really work as it would send things to the console which might not be operational under stack-overflow conditions.